openedx/adg/lms/applications/views.py

```diff
@@ -96,7 +96,8 @@ def get(self, request):
         Returns:
             HttpResponse object.
         """
-        user_application_hub = ApplicationHub.objects.get(user=self.request.user)
+        user = self.request.user
```

Comment:
Do we really need to change this? We can simply get the user from user_application_hub. And we can do this all in `get_application_hub_instructions `.